PR: https://github.com/odoo/odoo/pull/85199

From: 66a3a409eb36c6518859cc967a6f28cb4f70e0fa
From: Julien Castiaux
Date: 2022-02-23 12:59:52

Structural Changes: 3
Total Changes: 6

Revert "[FIX] event: remove `prefetch=True`"

This reverts commit 8dda03228e7482c7900144e697e2753188713899.

Part-of: odoo/odoo#85199

================================= pseudo patch: =================================

--- a/addons/event/models/event_stage.py
+++ b/addons/event/models/event_stage.py
@@ -17,11 +17,11 @@ class EventStage(models.Model):
         string='End Stage', default=False,
         help='Events will automatically be moved into this stage when they are finished. The event moved into this stage will automatically be set as green.')
     legend_blocked = fields.Char(
-        'Red Kanban Label', default=lambda s: _('Blocked'), translate=True, required=True,
+        'Red Kanban Label', default=lambda s: _('Blocked'), translate=True, prefetch=True, required=True,
         help='Override the default value displayed for the blocked state for kanban selection.')
     legend_done = fields.Char(
-        'Green Kanban Label', default=lambda s: _('Ready for Next Stage'), translate=True, required=True,
+        'Green Kanban Label', default=lambda s: _('Ready for Next Stage'), translate=True, prefetch=True, required=True,
         help='Override the default value displayed for the done state for kanban selection.')
     legend_normal = fields.Char(
-        'Grey Kanban Label', default=lambda s: _('In Progress'), translate=True, required=True,
+        'Grey Kanban Label', default=lambda s: _('In Progress'), translate=True, prefetch=True, required=True,
         help='Override the default value displayed for the normal state for kanban selection.')
